welcome.css

```diff
@@ -72,6 +71,82 @@ html, body {
 
 .text-tag {
   text-align: center;
-  font-size: 32px;
+  font-size: 38px;
   color: olive;
+}
+
+.burger-icon:before {
+  display: none;
+  font-size: 32px;
+}
+
+.click-icon {
+  display: none;
+}
+
+@media (max-width: 800px) {
```

Comment:
Please write a comment before this code for phone.
Similarly, please add the style when viewing on a computer. At the top of this CSS file

`Style when viewed on mobile`
`@media (max-width: 800px) {`